package.json

```diff
@@ -25,10 +25,29 @@
     "onLanguage:reason"
   ],
   "icon": "assets/logo.png",
+  "main": "./dist/extension.js",
   "contributes": {
     "configuration": {
       "title": "OCaml configuration",
       "properties": {
+        "ocaml.toolChainRoot": {
+          "type": "string",
+          "default": null,
+          "description": "Determines where to find the toolchain for a given project"
+        },
+        "ocaml.packageManager": {
+          "type": "string",
+          "default": null,
+          "description": "Determines what package manager to use for a given project"
+        },
+        "ocaml.refmt.width": {
+          "type": [
+            "number",
+            null
+          ],
+          "default": null,
+          "description": "Set the width of lines when formatting code with refmt"
+        },
```

Comment:
I don't think we need this.

Reply:
You are right. Thanks for catching this. This has been removed